### PR TITLE
Adding Lozenge element

### PR DESCRIPTION
**Pull request by AaronPatterson1**

### CODE REVIEW

File: packages/_site/src/pages/components/lozenge/index.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function LozengePage() {
         >
             <ComponentHero
                 title="Lozenge"
-                description=""
+                description="A colored, emphasized label encased in a rounded rectangle element."
```
Comment (clearfeld) [inline packages/_site/src/pages/components/lozenge/index.tsx]: Description that should be used:
"A lozenge is a visual indicator used to highlight an item's status for quick recognition."

File: packages/_site/src/pages/components/routes_list_docs_components.ts [hunk 2]
```diff
@@ -146,7 +146,7 @@ export const componentsList: I_ComponentsList[] = [
 
   {
     title: "Lozenge",
-    description: "",
+    description: "A colored, emphasized label encased in a rounded rectangle element.",
```
Comment (clearfeld) [inline packages/_site/src/pages/components/routes_list_docs_components.ts]: Same as above description.

File: packages/_site/src/pages/components/lozenge/index.tsx [hunk 3]
```diff
@@ -110,6 +110,465 @@ export default function LozengeExample() {
...
+                                    language="tsx"
+                                    code={`import React from 'react';
+
+import { Lozenge, LozengeVariants } from "@controlkit/lozenge";
+
+export default function LozengeExample() {
+	return (
+        <Lozenge
+            variant={LozengeVariants.DEFAULT}
+            isBold={true}
```
Comment (clearfeld) [inline packages/_site/src/pages/components/lozenge/index.tsx]: Don't use the isBold here.

File: packages/_site/src/pages/components/lozenge/index.tsx [hunk 4]
```diff
@@ -110,6 +110,465 @@ export default function LozengeExample() {
...
+}
+`}
+                                />
+                            </div>
+                        </div>
+
+                        <div>
+                            <ExampleBlock
+                                title="In Progress Bold"
+                                description="Blue Lozenged element, indicated removal or error, boldened."
```
Comment (clearfeld) [inline packages/_site/src/pages/components/lozenge/index.tsx]: A blue lozenge doesn't indicate removal or an error.

File: packages/_site/src/pages/components/lozenge/index.tsx [hunk 5]
```diff
@@ -110,6 +110,465 @@ export default function LozengeExample() {
...
+}
+`}
+                                />
+                            </div>
+                        </div>
+
+                        <div>
+                            <ExampleBlock
+                                title="Removed Bold"
+                                description="Red Lozenged element, indicated removal or error, boldened."
```
Comment (clearfeld) [inline packages/_site/src/pages/components/lozenge/index.tsx]: Lozenges don't indicate errors.

File: packages/_site/src/pages/components/lozenge/index.tsx [hunk 6]
```diff
@@ -110,6 +110,465 @@ export default function LozengeExample() {
...
+}
+`}
+                                />
+                            </div>
+                        </div>
+
+                        <div>
+                            <ExampleBlock
+                                title="Removed"
+                                description="Red Lozenged element, indicated removal or error."
```
Comment (clearfeld) [inline packages/_site/src/pages/components/lozenge/index.tsx]: Lozenges don't indicate errors.

**Review by clearfeld — CHANGES_REQUESTED** [review]

**Comment by AaronPatterson1** [comment]
Fixed, please check again.

**Review by clearfeld — APPROVED** [review]

**Pull request merged**